### PR TITLE
Added Wide and Deep Neural Network example

### PR DESCRIPTION
**Pull request by gaurav272333**

### CODE REVIEW

File: showcase/tabular/wide_and_deep.py [hunk 1]
```diff
@@ -0,0 +1,145 @@
...
+from fastestimator.network.loss import BinaryCrossentropy
+from fastestimator.estimator.trace import Accuracy
+from fastestimator.dataset import movielens20m_data
+
+_USER_INPUT_DIM = 138493
+_MOVIE_INPUT_DIM = 26744
+_USER_EMBEDDING_DIM = 64
+_MOVIE_EMBEDDING_DIM = 15
+
+GENRES = [
```
Comment (TortoiseHam) [inline showcase/tabular/wide_and_deep.py]: the should probably be in the data file

File: fastestimator/dataset/movielens20m_data.py [hunk 2]
```diff
@@ -0,0 +1,108 @@
...
+    # Creation of Train and Evaluation Sets
+    train_df = df.sample(frac=0.8, random_state=0)
+    eval_df = df.drop(train_df.index)
+
+    train_df = train_df.reset_index(drop=True)
+    eval_df = eval_df.reset_index(drop=True)
+
+    # Applying Standard Scaler
+    Std_cols = ["genres_count", "year", "month", "day"]
+    scaler = StandardScaler()
```
Comment (TortoiseHam) [inline fastestimator/dataset/movielens20m_data.py]: should we be demonstrating how to do this in Pipeline instead?

File: showcase/tabular/wide_and_deep.py [hunk 3]
```diff
@@ -0,0 +1,145 @@
...
+
+    # deep Part
+    x_deep1 = concatenate([embed_layer, input_genres, input_time])
+    x_deep2 = Dense(256, activation="relu")(x_deep1)
+    x_deep3 = Dense(128, activation="relu")(x_deep2)
+    x_deep4 = Dense(64, activation="relu")(x_deep3)
+
+    # wide part
+    x_wide1 = concatenate([input_genres, input_time])
+    x_wide2 = Dense(16, activation="relu")(x_wide1)
```
Comment (TortoiseHam) [inline showcase/tabular/wide_and_deep.py]: isn't this 'narrower' than x_deep1?

File: showcase/tabular/wide_and_deep.py (hunk 1 above)
Comment (gaurav272333) [reply showcase/tabular/wide_and_deep.py]: Sure. This makes sense. I will move this to the data file.

File: fastestimator/dataset/movielens20m_data.py (hunk 2 above)
Comment (gaurav272333) [reply fastestimator/dataset/movielens20m_data.py]: As this needs to be applied on entire dataset, we won't be able to apply this in Pipeline.

**Pull request closed without merging**